components/deferred-billing-terms.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { DeferredBillingTerms } from './deferred-billing-terms';
+
+export default {
+	title: 'Deferred Billing Terms',
+	component: DeferredBillingTerms,
+	argTypes: {
+		hasError: { control: 'boolean' },
+		isChecked: { control: 'boolean' },
+		isDeferredBilling: { control: 'boolean' }
+	}
+};
+
+export const Basic = (args) => <DeferredBillingTerms {...args} />;
+Basic.args = {};
+
+export const CheckedFlow = (args) => <DeferredBillingTerms {...args} />;
+CheckedFlow.args = {
+	isChecked: true,
+	isDeferredBilling: true,
+	hasError: false
+};
```

Comment:
I think we usually have a newline at end of file
```suggestion
};

```